server/chat-plugins/roomsettings.js

```diff
@@ -328,6 +328,10 @@ exports.commands = {
 			if (!this.can('editroom', null, room)) return;
 		} else if (room.battle) {
 			if (!this.can('editprivacy', null, room)) return;
+			if (room.battle) {
```

Comment:
The same condition is there 2 lines above.

Reply:
Thanks!